ansible/setup.yml

```diff
@@ -0,0 +1,80 @@
+#################################################
+# Initial Server Setup for Radicle Components
+#################################################
+---
+- hosts: all
+  vars:
+      ansible_python_interpreter: /usr/bin/python3
+      ansible_ssh_common_args: '-o StrictHostKeyChecking=no'
+      #RAD_SUBGRAPH: https://...
+      #RAD_RPC_URL: wss://...
+      #RAD_ORG: 0x...
+
+  tasks: 
+    - name: Create Directories
+      file:
+        path: "{{ item }}"
+        state: directory
+      with_items:
+      - /app/radicle/root
+
+    - name: Copy Org Identity File
+      copy:
+        src: ../keys/{{ RAD_ORG }}.key
```

Comment:
The org identity file is typically created by the org node on startup, if it doesn't exist. Did you have another plan for how this file should be created?

Reply:
This simplifies retry logic (on retry we already have the same key), and backups (already living on two servers). We can potentially do it in a pull way, but we need to wait for container to start, and wait for the file to appear.

Also, there's no profile directory, when it's generated on startup:

```
└── radicle
    ├── identity
    └── root
        ├── git
        │   ├── config
        │   ├── description
        │   ├── HEAD
        │   ├── hooks
        │   │   └── README.sample
        │   ├── info
        │   │   └── exclude
        │   ├── logs
        │   │   └── refs
        │   │       └── namespaces
        │   ├── objects
        │   │   ├── info
        │   │   └── pack
        │   └── refs
        │       ├── heads
        │       └── tags
        ├── git-includes
        └── keys
```

Reply:
Well, the profile directory is not necessary on the server, it's an end-user feature.

Reply:
Fixed https://github.com/radicle-dev/radicle-cloud/pull/1/commits/23b3fdf6afc44ae5bc92719075d587804cc1f533